### ISSUE TITLE
Can't change Map Type to Satellite View

### ISSUE THREAD

**Issue by el-saadi** [cyberhobo/wordpress-geo-mashup#863]
Hello,
i want to change the maptype of my global map to  Satellite view, but it always stays at the defaul Road/Normal view. 
In the documentation there are these parameters **map_type - G_NORMAL_MAP, G_SATELLITE_MAP, G_HYBRID_MAP, G_PHYSICAL_MAP**, but this won't work for me. Here is my code snippet of the global map in Template.

`    <?php $map_params = array(
                                        "map_content" => "global", 
                                        "map_type" => "G_SATELLITE_MAP",
                                        "height" => "800p", 
                                        "width" => "100%",
                                        "zoom" => '15',
                                        "map_control" => "GSmallZoomControl",
                                        "add_map_type_control" => array("G_NORMAL_MAP", "G_SATELLITE_MAP", "G_PHYSICAL_MAP"),
                                        "add_overview_control" => "true",
                                        "marker_select_highlight" => "true",
                                        "marker_min_zoom" => "true"
                                        );
       echo GeoMashup::map($map_params) ?>

![Codesnippet](https://user-images.githubusercontent.com/62426508/89317310-7b882f00-d67d-11ea-8a56-f7de72a77354.png)

What's the issue with my parameters, or is there a setting i have to made before hand? I hope you could help me with my problem.

`

**Comment by cyberhobo** [comment]
That looks right to me, could be a bug. Is your Map Provider set to Google?

**Comment by el-saadi** [comment]
Thank you for your fast answer. 
No, I use the Leaflet as my Map Provider.

**Comment by cyberhobo** [comment]
I can reproduce that bug with Leaflet, should be able to fix it in the next release.

**Comment by netzgestaltung** [comment]
Hi there,

i tested the update on my test environment and now the Leaflet map here always shows the satelite view at page loading.

i added the mentioned config

```
<?php echo GeoMashup::map( array(
    'name' => 'search-results-map',
    'search_text' => $search_text,
    'object_ids' => $geo_mashup_search->get_the_ID_list(),
    'center_lat' => $near_location['lat'],
    'center_lng' => $near_location['lng'],
    'search_lat' => $near_location['lat'],
    'search_lng' => $near_location['lng'],
    'map_content' => 'global',
    'object_name'=> $object_name,
    'zoom' => 	$approximate_zoom + 6, // Adjust to taste
    'map_type' => 'G_NORMAL_MAP',
  ) ); ?>```

but it shows the satelite view anyways.

maybe this is related but i cant estimate it.

on my live page with these options (wihtout map_type) on wihtout the update it shows the street map_type

**Comment by cyberhobo** [comment]
Thanks for checking, looks like I released a new bug in place of the old one: #865
